frontend/packages/dev-console/src/components/topology/components/nodes/GroupNode.tsx

```diff
@@ -18,7 +18,7 @@ const LEFT_MARGIN = 20;
 const TEXT_MARGIN = 10;
 
 const truncateOptions: TruncateOptions = {
-  length: 35,
+  length: 20,
```

Comment:
Noticed that in some cases the name of groups would overlap the group boundaries as well. Changes the truncation length to be the standard 20 characters.